core/src/main/java/org/apache/druid/data/input/impl/TimestampSpec.java

```diff
@@ -60,12 +65,14 @@ public TimestampSpec(
       @JsonProperty("column") String timestampColumn,
       @JsonProperty("format") String format,
       // this value should never be set for production data
-      @JsonProperty("missingValue") DateTime missingValue
+      @JsonProperty("missingValue") DateTime missingValue,
+      @JsonProperty("timeZone") String timeZone
   )
   {
     this.timestampColumn = (timestampColumn == null) ? DEFAULT_COLUMN : timestampColumn;
     this.timestampFormat = format == null ? DEFAULT_FORMAT : format;
-    this.timestampConverter = TimestampParser.createObjectTimestampParser(timestampFormat);
+    this.timeZone = (StringUtils.isBlank(timeZone)) ? DateTimes.UTC_TIMEZONE : timeZone;
```

Comment:
Maybe remove extra parenthesis on the condition?

Reply:
ok, I'll change it.